### DISCUSSION TITLE
Can't apply for student developer pack; asks me to complete my already complete profile.

### DISCUSSION THREAD

**Discussion by Kataneco** [community/community#42633] [GitHub Education]
I have already applied for and got the student pack before but it expired a few days ago, along with my CLion license ;(

So I tried re-applying with my school email and school ID but it wouldn't accept it and tell me to complete my GitHub user profile. Although I don't really like giving out my irl information away I edited my profile with my full name and stuff, retried, same error.
I also tried uploading a different form of proof (screenshot of school portal), but then it told me I **had to** use a dated school ID.

Similar threads to this have been created but it seems like their issues are just solved by some kind of divine intervention so I'm kind of lost here. 

What do I do now?

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159
